pipelines/nextflow/modules/genome_metadata/prepare_genome_metadata.nf

```diff
@@ -25,6 +25,6 @@ process PREPARE_GENOME_METADATA {
     shell:
     output_json = "genome.json"
     '''
-    prepare_genome_metadata --input_path !{input_json} --output_path !{output_json}
+    prepare_genome_metadata --input_file !{input_json} --output_file !{output_json}
```

Comment:
```suggestion
    prepare_genome_metadata --input_file !{input_json} > genome.json
```
Why are we not eliminating the `--output_file` here

Reply:
In this case the content is written by `prepare_genome_metadata()`, not `main()`, thus the parameter needs to be passed down. We could definitely update the code to use the same approach as in other instances. Shall I do it?

Reply:
No, its clear now so I am happy with it

Reply:
I like this suggestion, but will leave it for another PR that better suits such a change.